src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -3207,6 +3429,10 @@ int32_t NaClSysFork(struct NaClAppThread *natp) {
 
 fail:
 
+  #ifdef TRACING
+  NaClStraceFork(nap->cage_id, ret);
```

Comment:
this one too

Reply:
this is a valid (very important actually) syscall, we do want to trace this one

Reply:
making it

Reply:
this is being traced already, is the current structure correct?